Source/PINDiskCache.m

```diff
@@ -83,6 +85,12 @@ @implementation PINDiskCache
 
 #pragma mark - Initialization -
 
+- (void)dealloc
+{
+    __unused int result = pthread_mutex_destroy(&_mutex);
+    NSCAssert(result == 0, @"Failed to destroy lock in PINMemoryCache %p. Code: %d", (void *)self, result);
```

Comment:
Add `pthread_cond_destroy` here

Reply:
I knew there had to be something like that (but couldn't find docs)